tests/functional/lifecycle/LifecycleObjectProcessor.spec.js

```diff
@@ -19,15 +21,18 @@ const {
 
 werelogs.configure({ level: 'warn', dump: 'error' });
 
-describe('Lifecycle Object Processor', function lifecycleObjectProcessor() {
+const backbeatRoutes = ['deleteObject'];
+
+describe.only('Lifecycle Object Processor', function lifecycleObjectProcessor() {
```

Comment:
only here

Reply:
Its for testing purpose, need to be remove.. Should have set it in the PR desc mb